ui/lib/kv/addon/components/kv-delete-modal.hbs

```diff
@@ -0,0 +1,61 @@
+<button type="button" class="toolbar-link" {{on "click" (fn (mut this.modalOpen) true)}} data-test-kv-delete={{@mode}}>
+  {{yield}}
+</button>
+{{#if this.modalOpen}}
+  <Modal
+    @title={{this.modalDisplay.title}}
+    @onClose={{fn (mut this.modalOpen) false}}
+    @isActive={{this.modalOpen}}
+    @type={{this.modalDisplay.type}}
+    @showCloseButton={{true}}
+  >
+    <section class="modal-card-body">
+      <p class="has-bottom-margin-s">
+        {{this.modalDisplay.intro}}
+      </p>
+      {{#if (eq @mode "delete")}}
+        <div class="is-flex-column">
+          {{#each this.deleteOptions as |option|}}
+            <ToolTip @verticalPosition="above" @horizontalPosition="left" as |T|>
+              <T.Trigger @tabindex="-1">
+                <div class="is-flex-align-baseline has-bottom-margin-m">
+                  <RadioButton
+                    id={{option.key}}
+                    class="radio top-xxs"
```

Comment:
I noticed that the `top-xxs` class was added but would an existing margin or padding class achieve the same thing?

Reply:
I tried different margins and paddings, but they had no effect. The margin moves both the radio button and text and padding does nothing. Top was the only CSS I could figure out without doing a larger CSS change to nudge the radio buttons down a smidge.